account_e-invoice_generate/__manifest__.py

```diff
@@ -4,12 +4,14 @@
 
 {
     'name': 'Account e-invoice Generate',
-    'version': '11.0.1.0.0',
+    'version': '12.0.1.0.0',
     'category': 'Accounting & Finance',
     'license': 'AGPL-3',
     'summary': 'Technical module to generate PDF invoices with '
                'embedded XML file',
-    'author': 'Akretion,Odoo Community Association (OCA)',
+    'author': 'Akretion, '
+              'Shine IT<contact@openerp.cn>, '
```

Comment:
Changes you have made for migrating the module doesn't grant you co-authorship. Please remove this.